lib/restaurant.rb

```diff
@@ -0,0 +1,39 @@
+class Restaurant
+
+  attr_reader :menu
+
+  def initialize
+    @menu = []
+  end
+
+  def make_dish_available(dish)
+    @menu.push(dish)
+  end
+
+  def view_menu
+    fail "No dishes available on this menu" if no_dishes_available?
+    menu_string = ""
+    menu.each_with_index do |dish, index|
+      menu_string << "#{index+1}: #{dish.name} | Price: £#{'%.2f' % dish.price}\n"
```

Comment:
Metrics/LineLength: Line is too long. [82/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/FormatString: Favor `format` over `String#%`. (https://github.com/bbatsov/ruby-style-guide#sprintf)